src/libsystemd-network/sd-dhcp-client.c

```diff
@@ -166,6 +166,7 @@ static const uint8_t default_req_opts_anonymize[] = {
 };
 
 static const char* const dhcp_state_table[_DHCP_STATE_MAX] = {
+        [DHCP_STATE_STOPPED]              = "stopped",
```

Comment:
Hm? You may spuriously rebased? Please re-rebase on the current git HEAD. I guess this commit is not necessary.

Reply:
No, I defined dhcp_state_table and then reordered it in a subsequent commit that also previously incorporated your change for `DHCP_STATE_STOPPED` and `DHCP_STATE_INIT`. I completely unstacked the commits and built them up again so that there isn't any of this confusion. I apologize.